package.json

```diff
@@ -4,7 +4,9 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "nyc mocha --require @babel/register ./tests/*.js  --timeout 25000 --exit",
+    "start": "set NODE_ENV=production && babel-node ./src/index.js",
```

Comment:
For the production, Do you need to use the same folder for development? Just use the babel generated files!